app/src/main/java/org/sopt/daangnmarket_android/ui/viewmodel/WriteViewModel.kt

```diff
@@ -145,4 +161,66 @@ class WriteViewModel @Inject constructor(
         _selectedImageList.value = lastSelectedImageList
         Log.i("lastSelectedImageList", lastSelectedImageList.toString())
     }
-}
\ No newline at end of file
+
+    fun multipart(
+        title: String,
+        price: String,
+        content: String,
+        image: List<Bitmap>
+    ) {
+        val titleRequestBody =
+            title.toRequestBody("text/plain".toMediaTypeOrNull())
+        val priceRequestBody =
+            price.toRequestBody("text/plain".toMediaTypeOrNull())
+        val contentRequestBody =
+            content.toRequestBody("text/plain".toMediaTypeOrNull())
+
+        val requestBodyHashMap = HashMap<String, RequestBody>()
+        requestBodyHashMap["title"] = titleRequestBody
+        requestBodyHashMap["price"] = priceRequestBody
+        requestBodyHashMap["content"] = contentRequestBody
+
+        val imageListMultipartBody = mutableListOf<MultipartBody.Part>()
+
+        for (i in 0 until image.size) {
+            val imageRequestBody = BitmapRequestBody(image[i])
+            val imageMultipartBody: MultipartBody.Part =
+                MultipartBody.Part.createFormData(
+                    "image",
+                    "and" + System.currentTimeMillis().toString(),
+                    imageRequestBody
```

Comment:
```suggestion
                    "and" + System.currentTimeMillis().toString(),
                    imageRequestBody + i.toString()
```
* iOS와 중복을 피하기 위해 접두사 and 사용
* 1970.01.01 00:00과 현재 시간의 차이 사용
* 맨 끝에는 이미지의 인덱스 값을 접미사로 사용
이 3가지 방식을 통해 이미지의 중복이 없도록 처리하고자 하는 의도였습니다.